src/utils/selectors/index.ts

```diff
@@ -46,7 +44,12 @@ export const effectsInNgModuleProviders =
   `${ngModuleProviders} Identifier` as const
 
 export const actionDispatch = (storeName: string) =>
-  `ExpressionStatement > CallExpression:matches([callee.object.name='${storeName}'][callee.property.name='dispatch'], [callee.object.object.type='ThisExpression'][callee.object.property.name='${storeName}'][callee.property.name='dispatch'])` as const
+  `CallExpression:matches([callee.object.name='${storeName}'][callee.property.name='dispatch'], [callee.object.object.type='ThisExpression'][callee.object.property.name='${storeName}'][callee.property.name='dispatch'])` as const
+
+export const dispatchInEffects = (storeName: string) =>
+  `${createEffectExpression} ${actionDispatch(
+    storeName,
+  )} > MemberExpression` as const
```

Comment:
Instead of repeating the selectors, here I'm reusing the `createEffectExpression` and `actionDispatch`.